### DISCUSSION TITLE
can add multiple php?

### DISCUSSION THREAD

**Discussion by langziyang** [php/frankenphp#1089] [Q&A] [answered]
frankenphp can add multiple php?

**Comment by withinboredom** [comment]
Can you clarify your question? What do you mean by "multiple php"? Multiple php scripts? Multiple versions of php? Multiple sites?

**Reply by langziyang** [reply]
sorry about that,i mean multiple php version,

**Reply by withinboredom** [reply, marked as answer]
It currently supports PHP 8.2 and 8.3 (and tests are run for 8.4 release candidates). However, frankenphp runs PHP in threads (not separate processes like fpm), thus only can support a single version per process. So, if you want to run multiple versions of PHP on the same machine, you would have to run multiple frankenphp's compiled against different versions of PHP. Does that help?
